app/models/miq_report/seeding.rb

```diff
@@ -68,14 +68,15 @@ def sync_from_file(filename, dir, typ)
       rpt[:template_type] = typ
       rec = self.find_by_filename(rpt[:filename])
 
+      type_titleized = typ.titleize
       if rec
-        if rec.filename && (rec.file_mtime.nil? || rec.file_mtime.utc < rpt[:file_mtime])
-          _log.info("#{typ.titleize}: [#{rec.name}] file has been updated on disk, synchronizing with model")
+        if rec.filename && (rec.file_mtime.nil? || rec.file_mtime.utc.to_i < rpt[:file_mtime].to_i)
```

Comment:
+1 On calling round on the Time objects. Seems much cleaner to me. I'm good with the rest :+1: